.github/workflows/labeler.yml

```diff
@@ -1,12 +1,12 @@
 # https://github.com/actions/labeler
 name: "Pull Request Labeler"
 on:
-  - pull_request
+  - pull_request_target
 
 jobs:
   triage:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/labeler@master
+      - uses: actions/labeler@main
```

Comment:
We have no main (yet)?

Reply:
I think you mentioned it is not working great, should we remove, or does this fix things?